packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -0,0 +1,319 @@
+    disabled,
+    selected,
+    isOpen,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    padding: ${dropdown.selector.padding.top}px 
+      ${dropdown.selector.padding.right}px
+      ${dropdown.selector.padding.bottom}px
+      ${dropdown.selector.padding.left}px;
+
+    background-color: ${dropdown.selector.background};
+    border-radius: ${dropdown.selector.border.radius}px;
+    border: 1px solid ${dropdown.selector.border.color};
+    ${
+      disabled
+        ? `border-color: ${dropdown.disabled.selector.border.color};`
+        : ''
+    };
+    ${
+      selected
+        ? `border-color: ${dropdown.selected.selector.border.color};`
+        : ''
+    };
+    ${
+      isOpen && !disabled
+        ? `border-color: ${dropdown.hover.selector.border.color}`
+        : ''
+    }
+    &:hover{
+      ${
+        !disabled ? `border-color: ${dropdown.hover.selector.border.color}` : ''
+      };
+    }
+  `}
+`;
+
+const Input = styled.input`
+  ${({
+    disabled,
+    selected,
+    theme: {
+      yoga: {
+        baseFont,
+        components: { dropdown },
+      },
+    },
+  }) => `
+    width: 100%;
+    padding: 0;
+
+    background-color: transparent;
+    border: none;
+
+    font-family: ${baseFont.family};
+    font-size: ${dropdown.input.font.size}px;
+    line-height: ${dropdown.input.font.lineHeight}px;
+    cursor: ${disabled ? 'not-allowed' : 'pointer'};
+
+    &, &::placeholder {
+      color: ${dropdown.input.font.color};
+      ${selected ? `color: ${dropdown.selected.input.font.color};` : ''}
+      ${disabled ? `color: ${dropdown.disabled.input.font.color};` : ''}
+    }
+  `}
+`;
+
+const Button = styled.button`
+  ${({
+    disabled,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    display: flex;
+    align-content: center;
+    justify-content: flex-end;
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+    padding-right: ${dropdown.button.paddingRight}px;
+
+    border: none;
+    outline: none;
+    background-color: transparent;
+    cursor: ${disabled ? 'not-allowed' : 'pointer'};
+  `}
+`;
+
+const ArrowIcon = styled(Arrow)`
+  ${({
+    isOpen,
+    disabled,
+    selected,
+    theme: {
+      yoga: {
+        colors: { primary },
+        components: { dropdown },
+      },
+    },
+  }) => `
+    fill: ${dropdown.disabled.arrow.fill};
+    ${disabled ? `fill: ${dropdown.disabled.arrow.fill};` : ''};
+    ${selected ? `fill: ${primary[3]};` : ''};
+    transform: rotate(${isOpen ? '180deg' : '0'});
+  `}
+`;
+
+const OptionsList = styled.ul`
+  ${({
+    selected,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    position: absolute;
+    top: 46px;
+    z-index: 1;
+
+    width: 100%;
+    max-height: 140px;
```

Comment:
```suggestion
```

What do you think to have a fixed height to the dropdown theme property? and use it in the dropdown itself and the option item and option list.

so, in this case, you can use a calculation here and do something like this:

```suggestion
    max-height: ${dropdown.height * 4.5}px;
```

to show 4 items and half of the fifth.

You can use this at the `top` property too.

Reply:
Nice Tip! <3